great_expectations/data_context/data_context/base_data_context.py

```diff
@@ -316,12 +319,12 @@ def __init__(
         # NOTE - 20210112 - Alex Sherstinsky - Validation Operators are planned to be deprecated.
         if (
             "validation_operators" in self.get_config().commented_map
-            and self.config.validation_operators
+            and self._project_config.validation_operators
```

Comment:
Here I think we still want to use the public property `self.config` rather than the private - or is there a reason I'm not thinking of?

Reply:
done!